### DISCUSSION TITLE
[@types/bootstrap] dropdown options missing autoClose

### DISCUSSION THREAD

**Discussion by dicbrus** [DefinitelyTyped/DefinitelyTyped#55866] [Issues with a @types package]
Bootstrap 5 dropdown supports `autoClose: boolean | option` that is missing in @types/bootstrap
https://getbootstrap.com/docs/5.0/components/dropdowns/

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
I created PR to fix that here #55868
